docs/tool/js/views/map-view.js

```diff
@@ -932,6 +931,32 @@
                         value = (value === null | value == "null") ? ' Unknown' : value; // handles when data has "null" as a value
                         dl.append('dd').text(value)
                     }
+
+                    var currentNlihc = projectData.properties.nlihc_id;
+                    var subsidyContainerId = 'subsFor' + currentNlihc;
+                    var subsidyTableId = 'subsTableFor' + currentNlihc;
+                    controller.getData({
+                        name: currentNlihc + '_subsidy',
+                        url: "http://hiapidemo.us-east-1.elasticbeanstalk.com/api/project/" + currentNlihc + "/subsidies",
+                        callback: function(data){
+                            current.append('div')
+                                .attr('id', subsidyContainerId)
+                                .append('p')
+                                .text('Subsidies for this project');
+                            d3.select('#' + subsidyContainerId)
+                                .append('div').attr('id', subsidyTableId);
+                                                
+                            new D3Table('#' + subsidyTableId)
+                                .data(data.items)
+                                .columns([
+                                    {field:'poa_end', label:'Scheduled End Date', class:'value', html: function(d){return d}},
+                                    {field:'poa_start', label:'Start Date', class:'value', html: function(d){return d}},
+                                    {field:'program', label:'Program', class:'value', html: function(d){return d}}
+                                ])
+                                .create();
```

Comment:
If you click the same property multiple times, the `create()` function is run multiple times, adding additional tables to the div. 

The best way to fix this is probably to wrap the whole `showProjectPreview` function in an if statement that only runs the update if the newly selected project is not the same as the previous project. While we could do it at the table level, there could be other things added int he future that have the same problem and this would avoid them. 

OR we could change how the behavior works so that the existing project preview is ALWAYS faded out before bringing in the new content, even if you've re-selected the same building. That would also fix a usability problem of people who wondered why nothing happened when they clicked a project if it was the one already selected. The data binding around row 865 is what is causing it to change only if it is a new selection, so that solution starts there.